change: [OCA-1411] - Use Passbolt CE full name

tbaka opened the pull request:
## Description 📝
Updates Passbolt Community Edition Marketplace App naming to better comply with brand/trademark guidelines.

## Changes  🔄
List any change relevant to the reviewer.
- Adds full text 'Passbolt Community Edition'

## Target release date 🗓️
8/19/24

bnussman-akamai reviewed (commented):
The `one-click-apps.spec.ts` cypress test is failing.

I think it might be because the API returns `"label": "Passbolt One-Click"` but `oneClickAppsv2.ts` has `name: 'Passbolt Community Edition'` I think these need to match or something. 

Can't wait till we delete Linode Create v1 so we can make this stuff less silly

dwiley-akamai reviewed (commented):
> The `one-click-apps.spec.ts` cypress test is failing.
> 
> I think it might be because the API returns `"label": "Passbolt One-Click"` but `oneClickAppsv2.ts` has `name: 'Passbolt Community Edition'` I think these need to match or something.

Maybe the API has to return `Passbolt Community Edition One-Click`?

tbaka commented:
I believe it is correct that the name field in oneClickappsv2.ts has to match the literal label on the deployment script on the backend. I've added a new script with the correct label, and updated the ID in the PR.

mjac0bs commented on packages/manager/.changeset/pr-10778-changed-1723563275697.md:
```diff
@@ -0,0 +1,5 @@
+"@linode/manager": Changed
+---
+
+updates Passbolt CE naming ([#10778](https://github.com/linode/manager/pull/10778))
```
```suggestion
Update Passbolt CE naming in Marketplace ([#10778](https://github.com/linode/manager/pull/10778))
```
Thanks for the changeset, @tbaka. The team recently published some [best-practice guidelines for writing changesets in our developer docs](https://linode.github.io/manager/CONTRIBUTING.html#writing-a-changeset) to make them easier to remember. Formatting according to those guidelines saves us a little time when we make a final review and edit of the changelog before releasing.

mjac0bs reviewed (approved):
![Screenshot 2024-08-14 at 9 11 08 AM](https://github.com/user-attachments/assets/1a5f679e-ff4e-47f2-aef7-1fa75bf3a75a)

✅ Seeing the full name

dwiley-akamai reviewed (approved):
Seeing "Passbolt Community Edition" in Marketplace ✅
`one-click-apps.spec.ts` passes in CI now ✅
